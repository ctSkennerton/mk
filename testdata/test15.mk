awktarget:Sawk -f /dev/stdin:
	BEGIN{print "greetings from awk"}