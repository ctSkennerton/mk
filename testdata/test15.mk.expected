awktarget: BEGIN{print "greetings from awk"}
